.github/workflows/e2e-test.yml

```diff
@@ -92,15 +92,9 @@ jobs:
         run: kubectl config use-context k3d-kcp
       - name: Deploy Istio on KCP Cluster
         run: |
-          curl -L https://istio.io/downloadIstio | TARGET_ARCH=x86_64 sh -
-          chmod +x istio-$ISTIO_VERSION/bin/istioctl
-          mv istio-$ISTIO_VERSION/bin/istioctl /usr/local/bin
           istioctl install --set profile=demo -y
```

Comment:
why here remove download `istioctl` but still use istioctl?

Reply:
we do not need to download it twice